README.md

```diff
@@ -180,7 +180,6 @@ A GitHub conflict is when people make changes to the same area or line in a file
 | [P](#p) | [Q](#q) | [R](#r) | [S](#s) | [T](#t) | [U](#u) | [V](#v) | [W](#w) | [X](#x) | [Y](#y) | [Z](#z) |
 
 - ### **A**
```

Comment:
Please add this line back.
```suggestion
- ### **A**

```

Reply:
![image](https://user-images.githubusercontent.com/121809535/227770915-94b0ff5d-3af1-4cdb-994a-afbb188f580b.png)
I don't get it . It looks same,isn't it?